lib/db_connection.ex

```diff
@@ -452,6 +456,28 @@ defmodule DBConnection do
     pool.child_spec({conn_mod, opts})
   end
 
+  @doc """
+  Forces all connections in the pool to disconnect within the given interval.
+
+  Once this function is called, the pool will disconnect all of its connections
+  as they are checked in or as they are pinged. Checked in connections will be
+  randomly disconnected within the given time interval. Pinged connections are
+  immediately disconnected - as they are idle (according to `:idle_interval`).
+
+  If the connection has a backoff configured (which is the case by default),
+  disconnecting means an attempt at a new connection will be done immediately
+  after, without starting a new process for each connection. However, if backoff
+  has been disabled, the connection process will terminate. In such cases,
+  disconnecting all connections may cause the pool supervisor to restart
+  depending on the max_restarts/max_seconds configuration of the pool,
+  so you will want to set those carefully.
+  """
+  def disconnect_all(conn, interval, opts \\ []) when interval >= 0 do
```

Comment:
Do we need the `opts` parameter? It seems unused down the call stack and in the tests.
Or it's for consistency with the signature of the other public functions in the module?

Reply:
Yes, just in case it is needed in the future.